x/profile/keeper/coordinator.go

```diff
@@ -103,8 +103,11 @@ func (k Keeper) GetAllCoordinator(ctx sdk.Context) (list []types.Coordinator) {
 }
 
 // GetCoordinatorAddressFromID returns a coordinator address from its id
-func (k Keeper) GetCoordinatorAddressFromID(ctx sdk.Context, id uint64) string {
-	return k.GetCoordinator(ctx, id).Address
+func (k Keeper) GetCoordinatorAddressFromID(ctx sdk.Context, id uint64) (string, bool) {
+	if !k.HasCoordinator(ctx, id) {
```

Comment:
I may need to see if we can remove this method as well from Starport template

Reply:
Are you mean the `Has<Object>` method? I think we can put it together with the `Get<Object>` method, as the map type. What do you think?

E.g:
```golang
GetCoordinator(ctx sdk.Context, id uint64) (coord types.Coordinator, found bool)
```

Reply:
Yes I was thinking standardising the generated method list independently from the type. This is not a big priority